### PR TITLE
Add date on top right + subtotals per currency

### PR DESCRIPTION
**Pull request by mdietrichc2c**
This PR is to add 2 things to the credit summary report:
- the current date on the top right,
- subtotals by currency.

### CODE REVIEW

File: account_credit_control/report/credit_control_summary.html.mako [hunk 1]
```diff
@@ -205,6 +214,23 @@ tr.line {
...
+        if currency in balance_due_per_currency:
+            balance_due_per_currency[currency] += line.balance_due
+        else:
+            balance_due_per_currency[currency] = line.balance_due
+      %>
+%endfor
+      </table>
+      <br/>
+      <table class="list_table" style="width: 100%;">
+%for currency in balance_due_per_currency.keys():
```
Comment (yvaucher) [inline account_credit_control/report/credit_control_summary.html.mako]: Isn't it possible to use this in mako?

```
%for currency, balance_due in balance_due_per_currency.iteritems():
```

**Comment by yvaucher** [comment]
Just a tiny remark

**Comment by mdietrichc2c** [comment]
@yvaucher you're right; I changed the code accordingly.

**Comment by yvaucher** [comment]
:+1: Thanks

**Comment by guewen** [comment]
:+1:

**Pull request merged**